src/github.js

```diff
@@ -6,27 +6,37 @@ const docker = require('./docker')
 const mem = require('mem')
 const npm = require('./npm')
 
-const findFile = async (token, repo, fileName, initialPath = '') => {
+const findFile = async (token, repo, fileName) => {
+  const files = await findFiles(token, repo, fileName)
+  if (files.length === 1) {
+    return files[0]
+  } else if (files.length > 1) {
+    const filePaths = files.map(file => file.path)
+    console.error(`Too many matches found: ${JSON.stringify(filePaths, null, 2)}`)
+  }
+  return undefined
+}
+
+const findFiles = async (token, repo, fileName, initialPath = '') => {
   const items = await listRepoContents(token, repo, initialPath)
   
+  const matchingFiles = []
   const files = items.filter(item => item.type === 'file')
   for (let i = 0; i < files.length; i++) {
     const file = files[i]
     if (file.name === fileName) {
-      return file
```

Comment:
will line 28 ever run? If the condition is true it seems the block would return file and skip it. If not is will skip the block.

Reply:
I'm not sure I understand your question. The `return` is only in the old version of the code; the new one simply adds the file to the list of matchingFiles.